extension/lib/shopify/CollectionRepository.js

```diff
@@ -27,7 +27,21 @@ class ShopifyCollectionRepository {
    * @returns {Promise<ShopifyCollection[]>}
    */
   async list () {
+    const regExp = new RegExp(/([0-9])\w+/)
+    const rawCollections = await this._shopifyCollectionCommandFactory.createList().execute()
 
+    const productCountPromises = rawCollections.map(collection => {
+      const id = regExp.exec(Buffer.from(collection.id.toString(), 'base64').toString())[0]
```

Comment:
The ShopifyCollectionRepositoryCommandListGraphQL should take care of this as we interested of having the real (numeric id) scoped here.

Reply:
done